nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -37,27 +52,40 @@ def __init__(self, model: ov.Model):
         super().__init__(model)
         self._model = model.clone()
         self.name_to_node_mapping = {op.get_friendly_name(): op for op in self._model.get_ops()}
+        self._model_precision = ModelPrecision.FP32
```

Comment:
Even after MO we can have mixed precision when some subgraphs can be still in FP32 while most of the model is in FP16. I wonder why precision is so important on the model level.

Reply:
I found the answer below but still, I am not sure that we can judge based on the type of the first constant.

Reply:
ticket: 101738